content/02.author-sections.md

```diff
@@ -509,7 +509,15 @@ This is an improved version of my main figure in this interesting study [@doi:10
 
 
 <!-- Jason Miller -->
+### Jason E. Miller
 
+Hi, I'm a postdoctoral fellow from the [Ritchie lab](https://ritchielab.org) at the University of Pennsylvania. 
+
```

Comment:
You can remove the blank line here and between the sentences below.  That will create a new paragraph when the manuscript is rendered.